runner/cf-driver/prepare.sh

```diff
@@ -80,6 +80,15 @@ start_container () {
         --docker-image "$image_name"
     )
 
+    # Entrypoint & command aren't available w/o loading job res file
+    img_data=$(jq -rc '.image' "$JOB_RESPONSE_FILE")
+    container_entrypoint=$(echo "$img_data" | jq -r '.entrypoint | select(.)')
+    container_command=$(echo "$img_data" | jq -r '.command | select(.)')
+
+    if [ -n "$container_entrypoint" ] || [ -n "$container_command" ]; then
+        push_args+=('-c' "${container_entrypoint[@]}" "${container_command[@]}")
```

Comment:
[question] does this need to be quoted so that entrypoint and command are both associated with the `-c`?

Reply:
I don't think so? I think I could also write this like…
```suggestion
        push_args+=("-c ${container_entrypoint[@]} ${container_command[@]}")
```
…but I'm not sure the difference is anything more than stylistic, especially because this is going to end up as one big string at the end.

But I'm by no means absolutely positive that there isn't some nuance or convention that I'm unaware of here.

Reply:
I'm not sure there is, I just know that I debugged an issue a week or two ago that came down to there being a difference between `eval $@` and `eval "$@"` (or similar).

I _think_ it'll be fine, so it's probably worth merging and making sure to test with a variety of entrypoint/command combinations, including commands that use arguments like `-p` that are also accepted by `cf push`

Reply:
Ahhh yeah that makes sense, I do think I might need to do something different to handle that kind of case. Good thinking!

I'm thinking I'll merge this, do some local testing, confirm with a trial run and then submit a bugfix if necessary, because this same pattern is used for services. Sound good to you?